src/components/Navbar/components/CurrentLocationWidget/CurrentLocationWidget.tsx

```diff
@@ -0,0 +1,51 @@
+import { Paths } from 'enums/Paths';
+import { useLocation } from 'react-router-dom';
+import { CurrentLocationWidgetDesktop } from './components/CurrentLocationWidgetDesktop';
+import { CurrentLocationWidgetMobile } from './components/CurrentLocationWidgetMobile';
+import { Container, WeatherIcon, Description, DegreeSign } from './CurrentLocationWidget.styled';
+
+const CurrentLocationWidget = () => {
+  const location = useLocation();
+  let isLoading = true;
+  const data: any = {}; // temporary solution in order not to use
+
+  const renderContent = () => {
+    if (location.pathname === Paths.Home) {
```

Comment:
**question** What do you think about replacing this with this solution in the navbar
 `{location.pathname !== Paths.Home && <CurrentLocationWidget />}`

Reply:
It is also a good idea, i will change it